services/report/__init__.py

```diff
@@ -936,17 +941,18 @@ def build_report_from_raw_content(
 
         log.debug("Retrieved report for processing from url %s", archive_url)
         try:
-            with metrics.timer(f"{self.metrics_prefix}.process_report") as t:
```

Comment:
May be out of the loop, but we're no longer using these metrics?

Reply:
I think tracing is a much better alternative to this. Also, I haven’t seen any dashboards actually using these metrics?